services/blockchain-connector/tests/constants/blocks.js

```diff
@@ -46,47 +46,171 @@ const block = {
 const blockWithTransaction = {
 	header: {
 		version: 2,
-		timestamp: 1649855393,
+		timestamp: 1668685948,
```

Comment:
Why are we updating all the fixtures?

Reply:
just the ones related to decodeTransaction / decodeBlock

Reply:
Applied.